xbmc/interfaces/Builtins.cpp

```diff
@@ -1843,14 +1847,28 @@ int CBuiltins::Execute(const std::string& execString)
     CSettings::Get().SetBool("debug.showloginfo", !debug);
     g_advancedSettings.SetDebugMode(!debug);
   }
+  //TODO deprecated. To be replaced by pvr.startmanager
   else if (execute == "startpvrmanager")
   {
     g_application.StartPVRManager();
   }
+  else if (execute == "pvr.startmanager")
```

Comment:
why not combine this condition with the previous one (same for `pvr.stopmanager`)?